src/com_tjvendors/admin/tjvendorsTemplate.json

```diff
@@ -134,27 +176,41 @@
     "client": "com_tjvendors",
     "key": "createVendorMailToOwnerWaitingForApproval",
     "title": "Vendor Creation email to Vendor Owner (approval On)",
-    "email_body": "<p>Dear {vendorer.name},<br /><br />You have created a new vendor <strong>{info.vendor_title}</strong>. Your vendor profile is under going approval process. Once your administrator approves you can proceed further.<br />Please wait for approval email.<br /><br />This is system generated email, please do not reply. Many thanks from {info.sitename}</p>",
-    "sms_body": "",
-    "push_body": "",
-    "web_body": "",
-    "email_subject": "Your vendor registration for \"{info.vendor_title}\" at {info.sitename} is pending for approval",
-    "sms_subject": "",
-    "push_subject": "",
-    "web_subject": "",
-    "email_status": 1,
-    "sms_status": 0,
-    "push_status": 0,
-    "web_status": 0,
     "user_control": 1,
+    "state": 1,
     "core": 1,
+    "email":{
+		"state": 1,
+		"cc": "",
+		"bcc": "",
+		"from_name": "",
+		"from_email": "",
+		"emailfields": {
+			"emailfields0": {
+				"id": "",
+				"language": "*",
+				"subject": "Your vendor registration for \"{{info.vendor_title}}\" at {{info.sitename}} is pending for approval",
+				"body": "<p>Dear {{vendorer.name}},<br /><br />You have created a new vendor <strong>{{info.vendor_title}}</strong>. Your vendor profile is under going approval process. Once your administrator approves you can proceed further.<br />Please wait for approval email.<br /><br />This is system generated email, please do not reply. Many thanks from {{info.sitename}}</p>"
+			}
+		}
+	},
+	"sms": {
+		"state": 1,
+		"smsfields": {
+			"smsfields0": {
+				"id": "",
+				"language": "*",
+				"body": "Dear {{vendorer.name}},You have created a new vendor {{info.vendor_title}}. Your vendor profile is under going approval process. Once your administrator approves you can proceed further. Please wait for approval email."
```

Comment:
change as


Dear {{vendorer.name}}, Your newly created vendor profile {{info.vendor_title}} is pending for approval. Wait for approval email  to proceed further.